tools/runners/test262-harness.py

```diff
@@ -495,19 +225,43 @@ def __init__(self, suite, name, full_path, strict_mode, command_template, module
         self.name = name
         self.full_path = full_path
         self.strict_mode = strict_mode
-        with open(self.full_path, "rb") as file_desc:
-            self.contents = file_desc.read()
-        test_record = parse_test_record(self.contents, name)
-        self.test = test_record["test"]
-        del test_record["test"]
-        del test_record["header"]
-        test_record.pop("commentary", None)    # do not throw if missing
-        self.test_record = test_record
         self.command_template = command_template
         self.module_flag = module_flag
-
+        self.test_record = {}
+        self.parse_test_record()
         self.validate()
 
+    def parse_test_record(self):
+        with open(self.full_path, "rb") as file_desc:
+            full_test = file_desc.read()
+
+        match = TEST_RE.search(full_test)
+        header = match.group('header')
+        self.test = match.group('test1') + match.group('test2')
+
+        match = YAML_INCLUDES_RE.search(header)
+        if match:
+            self.test_record["includes"] = [inc.strip() for inc in match.group('includes').split(',') if inc]
```

Comment:
You should use either apostrophe or double quotation mark characters for string literals.